docs/usage.md

```diff
@@ -22,12 +22,14 @@ We can open a virtual representation of this file using {py:func}`open_virtual_d
 
 ```python
 from virtualizarr import open_virtual_dataset
+from virtualizarr.parsers import HDFParser
+from obstore.store import LocalStore
 
-vds = open_virtual_dataset('air.nc')
+vds = open_virtual_dataset('air.nc', object_store=LocalStore, parser=HDFParser())
```

Comment:
context manager?

also, a few lines above, I suggest using a context manager for opening the air_temperature tutorial dataset